chain/types/ethtypes/eth_legacy_155_transactions.go

```diff
@@ -278,9 +280,9 @@ func deriveEIP155ChainId(v big.Int) big.Int {
 	return big.Div(v, big.NewInt(2))
 }
 
-func calcEIP155TxSignatureLen(chain uint64) int {
+func calcEIP155TxSignatureLen(chain uint64, v int) int {
```

Comment:
nit: for clarity I'd call this "parity" since it's not v but parity of y value corresponding to r.

Reply:
nah I'm wrong